lessons/231/python-app/main.py

```diff
@@ -115,6 +115,8 @@ async def create_device(
                 "updated_at": now,
             }
 
+            # Transaction ends here - automatic commit if successful, rollback if error
```

Comment:
This is fine but is it returning the connection to the connection pool. I'm nagging about this as there's a possibility to limit the whole application's concurrency to just the `POSTGRES_POOL_SIZE`.
Still there doesn't seem to be a transaction on the GO's side